python/triton/compiler/compiler.py

```diff
@@ -162,12 +183,13 @@ def _add_external_libs(mod, libs):
     add_external_libs(mod, list(libs.keys()), list(libs.values()))
 
 
-def ttgir_to_llir(mod, extern_libs, arch, tma_infos, waves_per_eu=0):
+def ttgir_to_llir(mod, extern_libs, target, tma_infos, waves_per_eu=0):
+    print("ttgir_to_llir")
```

Comment:
@jayfurmanek don't forget to remove the print

Reply:
good catch- I'll rebase/squash some of these extra commits as well